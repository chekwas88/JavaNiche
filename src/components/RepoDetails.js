/* eslint-disable prettier/prettier */
import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

const RepoDetails = ({repos, starredRepos}) => {
  return (
    <View style={styles.repoContainer}>
      <View style={styles.repoChildren}>
        <Text style={styles.repoTextBold}>
          Repo:  <Text style={styles.repoTextFaint}>{repos}</Text>
        </Text>
        <Text style={styles.repoTextBold}>
          StarredRepos:  <Text style={styles.repoTextFaint}>{starredRepos}</Text>
        </Text>
      </View>
    </View>
  );
};

const styles = StyleSheet.create({
  repoContainer: {
    width: '100%',
    height: 30,
    marginTop: '10%'
  },
  repoChildren: {
    flex: 1,
    flexDirection: 'row',
    justifyContent: 'space-around'
  },
  repoTextBold: {
    fontSize: 16,
    color: '#000',
    fontWeight: 'bold'
  },
  repoTextFaint: {
    fontSize: 14,
    color: '#8492A6'
  }
});

export default RepoDetails;
