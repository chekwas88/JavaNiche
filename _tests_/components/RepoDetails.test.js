import React from 'react';
import { shallow } from 'enzyme';
import RepoDetails from '../../src/components/RepoDetails';

describe('Test for RepoDetails component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<RepoDetails />);
    expect(
      wrapper
        .find('View')
        .find('View')
        .find('Text')
        .exists()
    ).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
