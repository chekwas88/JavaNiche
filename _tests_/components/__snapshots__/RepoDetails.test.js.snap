// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for RepoDetails component should render correctly 1`] = `ShallowWrapper {}`;
